From d822103336b17d40f5e93ab441e3cbf5ad000ee0 Mon Sep 17 00:00:00 2001
From: Michael Dickens <mlk@alum.mit.edu>
Date: Mon, 24 Mar 2014 18:02:41 -0400
Subject: [PATCH 07/13] When specifying a const reference member variable in
 the constructor's initialization list, the argument
 must also be a const reference.

---
 gnuradio-runtime/include/gnuradio/rpcserver_aggregator.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnuradio-runtime/include/gnuradio/rpcserver_aggregator.h b/gnuradio-runtime/include/gnuradio/rpcserver_aggregator.h
index cc19d7f..7390b52 100644
--- a/gnuradio-runtime/include/gnuradio/rpcserver_aggregator.h
+++ b/gnuradio-runtime/include/gnuradio/rpcserver_aggregator.h
@@ -50,7 +50,7 @@ private:
   template<class T, typename Tcallback>
   struct registerConfigureCallback_f: public std::unary_function<T,void>
   {
-    registerConfigureCallback_f(const std::string &_id,  const Tcallback _callback)
+    registerConfigureCallback_f(const std::string &_id,  const Tcallback& _callback)
       : id(_id), callback(_callback)
     {;}
 
@@ -72,7 +72,7 @@ private:
   template<class T, typename Tcallback>
   struct registerQueryCallback_f: public std::unary_function<T,void>
   {
-    registerQueryCallback_f(const std::string &_id,  const Tcallback _callback)
+    registerQueryCallback_f(const std::string &_id,  const Tcallback& _callback)
       : id(_id), callback(_callback)
     {;}
 
-- 
1.7.10.4

